src/System.Composition.Hosting/dir.props

```diff
@@ -0,0 +1,8 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+  <PropertyGroup>
+    <Version>1.0.31</Version>
```

Comment:
If we don't jump this one we should at least increment the minor (assuming we shipped it already which I'm not sure of).